### DISCUSSION TITLE
User script errors can't be read

### DISCUSSION THREAD

**Discussion by MattOslin** [foxglove/community#689] [Q&A]
They go off the edge of the screen, and long messages simply cannot be read. Copying the text you only get what is visible on screen.
<img width="679" alt="Screen Shot 2023-04-12 at 6 00 02 PM" src="https://user-images.githubusercontent.com/16124164/231618777-fb883753-9d59-41f1-ab4c-4b50a0ab7675.png">

**Comment by defunctzombie** [comment]
This sounds like a bug to me so I created: https://github.com/foxglove/studio/issues/5770 please fill out any useful reproduction steps.
